### ISSUE TITLE
Context timeout when saving lots of changes at once

### ISSUE THREAD

**Issue by mortenlj** [nais/console#87]
..also, no logs that could explain why it timed out

**Comment by sechmann** [comment]
Looks like the frontend is sending multiple queries in the same requests, but backend closes context after first is done?

```{"action":"graphql-api:team:add-owner","actor":"lwe@ssb.no","correlation_id":"8c17163f-8db7-11ed-956d-6e5413c2b607","level":"info","msg":"Add team owner: \"mmj@ssb.no\"","system":"graphql-api","target_type":"team","team":"platform","time":"2023-01-06T11:44:58.817062814Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:45:58.646841909Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:45:58.647460547Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:45:58.648029862Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:45:58.64891117Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:45:58.649186012Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:45:58.650975435Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:45:58.651169364Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:45:58.651718219Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:45:58.783868886Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:13.321644748Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:46:13.321916862Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:13.324710897Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:13.324931413Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:13.518414921Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:20.588853066Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:20.58928182Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:46:20.590950745Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:46:20.592886476Z"}
{"error":"timeout: context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:20.721679365Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:22.158908169Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:46:22.160273965Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:46:22.160716168Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:46:22.162335988Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:46:22.164337545Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:22.276855794Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:22.312070324Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:22.409550666Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:22.410865097Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:22.516431104Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:24.115350548Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:24.118434041Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:24.119215073Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:24.119532638Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:24.119700841Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:46:24.119931302Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:46:24.119941351Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:46:24.120634451Z"}
{"level":"error","msg":"unhandled error: input: addTeamOwners timeout: context already done: context canceled","system":"graphql-api","time":"2023-01-06T11:46:24.120758498Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:24.246247175Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:24.251055988Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:24.301991273Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:24.307495545Z"}
{"error":"context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:24.309952193Z"}
{"error":"timeout: context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:24.316158734Z"}
{"error":"timeout: context canceled","level":"error","msg":"get team","system":"graphql-api","team":"platform","time":"2023-01-06T11:46:24.409910025Z"}
```

**Comment by christeredvartsen** [comment]
Seems to have been fixed by 8a131364b590cece37f4733e1200cf3b0c531e6c.
